File: ios/RCTExampleModule.m
//
//  RCTExampleModule.m
//  nativemodules
//
//  Created by Daniel Sousa on 18/07/25.
//

#import "React/RCTBridgeModule.h"

@interface RCT_EXTERN_MODULE(RCTExampleModule, NSObject)
  RCT_EXTERN_METHOD(printMessage:(NSString *) message)
@end

File: ios/RCTExampleModule.swift
//
//  RCTExampleModule.swift
//  nativemodules
//
//  Created by Daniel Sousa on 18/07/25.
//

import Foundation

@objc (RCTExampleModule)
class RCTExampleModule: NSObject {
  
  @objc
  func printMessage(_ message: String) {
    print(message)
  }
}
